src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -7,7 +7,11 @@
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
-    <NoWarn>$(NoWarn);CS8002</NoWarn>
+    <!--
+      CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'
```

Comment:
Can we log an issue for this to get resolved?

Reply:
Oh, since this is the Dashboard perhaps this is fine to not have tripple slash docs here.

Reply:
Should we instead just set SkipArcadeNoWarnCS1591 to false or clear it out for this project and for the config one?

Reply:
> Oh, since this is the Dashboard perhaps this is fine to not have tripple slash docs here.

Correct. We aren't publishing this as a library package anymore.

> Should we instead just set SkipArcadeNoWarnCS1591 to false or clear it out for this project and for the config one?

We can't because of the way Arcade does MSBuild property defaulting. SkipArcadeNoWarnCS1591 needs to be set before you import the arcade .props file.

https://github.com/dotnet/arcade/blob/b372abae012cab2dbf011bd39c395e940e35da86/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L95